BLAC-369: update logos & text in explore

paulhagon opened the pull request.

yetti commented:
@paulhagon I fixed the tests and moved the 'Check eResources and Research Guides' outside the `<ul>` tag since you commented out the `<li>` tag.

yetti commented:
@paulhagon This is deploying to Devel and Test.

paulhagon commented:
> @paulhagon I fixed the tests and moved the 'Check eResources and Research Guides' outside the `<ul>` tag since you commented out the `<li>` tag.

Thanks - oops - I meant to uncomment those. I'm checking with Ella if I can remove them.
